tests/functional/cylc-combination-scripts/02-vr-restart.t

```diff
@@ -21,15 +21,14 @@
 
 
 . "$(dirname "$0")/test_header"
-set_test_number 6
+set_test_number 7
```

Comment:
This test appears to pass on master?

Reply:
Yes - turns out that in it's current state it is garbage.